docs/administrator-guide/configuration/README.md

```diff
@@ -3,48 +3,75 @@ order: 3
 ---
 # Configuration
 
-Kestra offers a lot of configuration options in order to customize the way you use it.
+Kestra offers a lot of configuration options and customization.
 
 How to pass those configuration options depends on how you deploy Kestra. Check the [deployment](/docs/administrator-guide/deployment/README.md) section related to you deployment mode for more information.
 
-## Kestra configuration 
-First things first, you will need to configure the queue type and the repository type.
+## Kestra internal components configuration 
 
-### Kafka configuration  
-To enable Kafka Queue, you need to add the following to your configuration files:
+Kestra has three internal components that must be configured:
+- The Internal Storage.
+- The Queue.
+- The Repository.
+
+### Internal storage configuration
+
+Kestra supports multiple internal storage types, the default being the local storage that will store data inside a local folder on the host filesystem. **Only suitable for local testing** as it didn't provide resiliency nor redundancy.
+
+To choose another storage type, you will need to configure the `kestra.storage.type` option, be sure to download the corresponding plugins first. The following example configure [Google Cloud Storage](./storage/README.md#gcs) for the internal storage.
```

Comment:
```suggestion
To choose another storage type, you will need to configure the `kestra.storage.type` option, be sure to download the corresponding plugins first. The following example configures [Google Cloud Storage](./storage/README.md#gcs) for internal storage.
```

Reply:
I use "the" as it's "the internal storage" of Kestra, one of its component.